test/utils/signPaymentChannelClaim/index.ts

```diff
@@ -0,0 +1,21 @@
+import {assertResultMatch} from '../../utils'
+import {channel, amount} from '../../fixtures/requests/sign-payment-channel-claim.json'
```

Comment:
you should just hardcode in the channel and amount, this is more confusing than anything else